src/storage/src/generated/convert/type/.sidekick.toml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[general]
+language             = 'rust'
+specification-source = 'google/type'
+
+[source]
+include-list = 'date.proto'
```

Comment:
comment: Ah, this is the only proto that storage needs from `google/type/`.

https://github.com/googleapis/googleapis/blob/7e9433ad5d5f8615648f25deab17bef31a6fcc36/google/storage/v2/storage.proto#L30